addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -67,18 +67,25 @@
 
     <!-- Media Dialog (allows to choose an img/pictogram/video) -->
     <div t-name="wysiwyg.widgets.media">
-        <ul class="nav nav-tabs" role="tablist">
-            <li class="nav-item"><a role="tab" href="#editor-media-image" aria-controls="editor-media-image" data-toggle="tab" t-attf-class="nav-link#{widget.noImages ? ' d-none' : ''}">Image</a></li>
-            <li class="nav-item"><a role="tab" href="#editor-media-document" aria-controls="editor-media-document" data-toggle="tab" t-attf-class="nav-link#{widget.noDocuments ? ' d-none' : ''}">Document</a></li>
-            <li class="nav-item"><a role="tab" href="#editor-media-icon" aria-controls="editor-media-icon" data-toggle="tab" t-attf-class="nav-link#{widget.noIcons ? ' d-none' : ''}">Pictogram</a></li>
-            <li class="nav-item"><a role="tab" href="#editor-media-video" aria-controls="editor-media-video" data-toggle="tab" t-attf-class="nav-link#{widget.noVideos ? ' d-none' : ''}">Video</a></li>
-        </ul>
+        <nav>
+            <div class="nav nav-tabs" role="tablist">
```

Comment:
The style of those tabs changed (the below border is not removed / the color of the text changed / ...). So it is either:
- Broken bootstrap structure
- Broken custom scss 
- Dead-code custom scss

Reply:
So the problem is that BS `nav-tabs` does not support the structure without `li` due to css rules.

Reverting to `li` for now.